build/cargo.gni

```diff
@@ -156,9 +156,13 @@ template("cargo_build") {
 
     if (is_mac) {
       args += [ "--mac_deployment_target=" + mac_deployment_target ]
-    } else if (is_ios) {
+    }
+
+    if (is_ios) {
```

Comment:
Can `is_mac` and `is_ios` both be true at the same time?

Reply:
I also have the same questions. It seems like this change here is not necessary.

Reply:
They shouldn't ever be true at the same time, there is a separate `is_apple` condition to check if you wanted either mac or ios

Reply:
> It seems like this change here is not necessary

I split it here so that I can use `if (is_ios) { ... } else { ... }`. Otherwise the `else` case will not catch macOS builds.